### ISSUE TITLE
Should AAR's pom.xml have dependencies declared?

### ISSUE THREAD

**Issue by pavlo** [chrisdoyle/gradle-fury#20]
Hey! 

I cloned the repo and ran `gradle build publishToMavenLocal` in order to see how it works and whether it fits my needs or not. It installed the artefacts into local repo well though I discovered that `hello-world-aar`'s pom.xml file did not have dependencies specified. Is it so intentionally? Is it something that is okay for an AAR? 

Wondering because there are dependencies for `hello-world-lib` module which a JAR file... I expected the AAR to have it too?

Cheers, Pavlo

**Comment by chrisdoyle** [comment]
@pavlo,

That is not intentional -- and looks to be a major oversight on my behalf!  At one point I had this working with the dependencies, but it must have got lost in the mix somewhere.  Look for a patch in the coming days as time permits here...

Thanks for bringing this to my attention,

--Doyle

**Comment by chrisdoyle** [comment]
@pavlo,

I have just committed a WIP branch to address this problem.  It is partially functional; it does not include the type or classifier properties yet.  A few more hours fighting Gradle with hopes to wrap this one up...

**Comment by chrisdoyle** [comment]
@pavlo,

I'm still trying to determine the best way to accurately capture both `<scope />` and `<classifier />` values in the `<dependency />` declarations.  No dice so far.  I found a rather disheartening post related to the subject:

https://discuss.gradle.org/t/maven-publish-plugin-generated-pom-making-dependency-scope-runtime/7494

I am attempting Mr. Niederwieser's "Use the 'pom.withXml' hook to fix the dependency scopes as needed" method as a "solution".  How important are `<scope />` and `<classifier />` to you?  Could I proceed forward with the current patch to get you over the hump?
